src/deephaven_ib/__init__.py

```diff
@@ -0,0 +1,894 @@
+        self._client = IbTwsClient(download_short_rates=download_short_rates)
+        self._dtw_requests = None
+        self._tables_raw = {f"raw_{k}": v for k, v in self._client.tables.items()}
+        self._tables = dict(sorted(IbSessionTws._make_tables(self._tables_raw).items()))
+
+    ####################################################################################################################
+    ####################################################################################################################
+    ## Connect / Disconnect / Subscribe
+    ####################################################################################################################
+    ####################################################################################################################
+
+    def connect(self, host: str = "", port: int = 7497, client_id: int = 0) -> None:
+        """Connect to an IB TWS session.  Raises an exception if already connected.
+
+        Args:
+            host (str): The host name or IP address of the machine where TWS is running. Leave blank to connect to the local host.
+            port (int): TWS port, specified in TWS on the Configure>API>Socket Port field.
+                By default production trading uses port 7496 and paper trading uses port 7497.
+            client_id (int): A number used to identify this client connection.
+                All orders placed/modified from this client will be associated with this client identifier.
+
+                Note: Each client MUST connect with a unique clientId.
+
+        Returns:
+              None
+
+        Raises:
+              Exception
+        """
+
+        self._client.connect(host, port, client_id)
+
+    def disconnect(self) -> None:
+        """Disconnect from an IB TWS session.
+
+        Returns:
+            None
+        """
+
+        self._client.disconnect()
+        self._dtw_requests = None
+
+    def is_connected(self) -> bool:
+        """Is there a connection with TWS?"""
+
+        return self._client.isConnected()
+
+    def _assert_connected(self):
+        """Assert that the IbSessionTws is connected."""
+
+        if not self.is_connected():
+            raise Exception("IbSessionTws is not connected.")
+
+    ####################################################################################################################
+    ####################################################################################################################
+    ## General
+    ####################################################################################################################
+    ####################################################################################################################
+
+    # TODO: fix Any type annotation with Deephaven v2
+    @staticmethod
+    def _make_tables(tables_raw: Dict[str, Any]) -> Dict[str, Any]:
+        def annotate_ticks(t):
```

Comment:
I made this static, because I generally avoid non-static methods in constructors unless I have a good reason.  I couldn't convince myself of a good reason to make it non-static.